### PR TITLE
Enable debugging for qubes-restore

### PR DESCRIPTION
**Pull request by alimirjamali**
Related: https://github.com/QubesOS/qubes-issues/issues/9387

Also adding option to go to next page on Enter in password field

### CODE REVIEW

File: qubesmanager/restore.py [hunk 1]
```diff
@@ -38,6 +39,17 @@
 
 
 # pylint: disable=too-few-public-methods
+def parse_args() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+            '--log', action='store', default='WARNING',
```
Comment (marmarek) [inline qubesmanager/restore.py:45]: Default was "INFO" before, changing to "WARNING" will hide most progress info...

**Review by marmarek — CHANGES_REQUESTED** [review]

File: qubesmanager/restore.py (hunk 1 above)
Comment (alimirjamali) [reply qubesmanager/restore.py]: Done

File: qubesmanager/restore.py [hunk 2]
```diff
@@ -38,6 +39,17 @@
 
 
 # pylint: disable=too-few-public-methods
+def parse_args() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+            '--log', action='store', default='INFO',
+            choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+            help='Provide logging level. Values: DEBUG, INFO, '
+            'WARNING (default), ERROR, CRITICAL')
```
Comment (marmarek) [inline qubesmanager/restore.py:48]: help needs update too

Comment (alimirjamali) [reply qubesmanager/restore.py]: > help needs update too

It is done.

**Comment by alimirjamali** [comment]
@marmarek I want to add (most of) the equivalent of command line options of the CLI backup & restore to their GUI counterparts. Should I do it in this PR or make a new PR? It is currently pending the decision on this PR.

**Comment by marmarek** [comment]
> @marmarek I would like to suggest a PR for adding (most of) the equivalent of command line options of the CLI backup & restore to their GUI counterparts. Should I do it in this PR or make a new PR? It is currently pending the decision on this PR.

a new one

**Review by marmarek — APPROVED** [review]

**Pull request merged**